static/js/components/search/LearnerResult_test.js

```diff
@@ -137,6 +143,21 @@ describe('LearnerResult', () => {
     });
   });
 
+  it('should pass down a function that can open a dialog to email a learner', () => {
+    helper.store.dispatch(setLearnerChipVisibility(USER_PROFILE_RESPONSE.username));
+    let openLearnerEmailComposerStub = helper.sandbox.stub();
+
+    let result = renderLearnerResult({openLearnerEmailComposer: openLearnerEmailComposerStub});
+    return helper.listenForActions([SET_LEARNER_CHIP_VISIBILITY], () => {
+      result.find(".learner-name").props().onMouseEnter();
+    }).then(() => {
+      // Execute the openLearnerEmailComposer function passed down to the LearnerChip
+      result.find("LearnerChip").props().openLearnerEmailComposer();
```

Comment:
Can you stick in an assertion, to be sure that we find the LearnerChip? Something like this:
```js
const chip = result.find("LearnerChip");
assert.lengthOf(chip, 1);
chip.props().openLearnerEmailCompose();
```

The way this is currently written, if the LearnerChip is not rendered, you'll get a misleading error message about calling `.props()` on nothing. This will make that failure case more clear.